### PR TITLE
Upgrade plugin and dependency versions

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -3,8 +3,8 @@ resolvers +=  Resolver.url(
   new java.net.URL("https://dl.bintray.com/meetup/sbt-plugins/")
 )(Resolver.ivyStylePatterns)
 
-addSbtPlugin("com.meetup" % "sbt-plugins" % "0.3.26")
+addSbtPlugin("com.meetup" % "sbt-plugins" % "0.3.38")
 
-addSbtPlugin("com.eed3si9n" % "sbt-dirty-money" % "0.1.0")
```
Comment (chenrui333) [inline project/plugins.sbt:8]: https://github.com/sbt/sbt-dirty-money#setup

File: project/plugins.sbt [hunk 2]
```diff
@@ -3,8 +3,8 @@ resolvers +=  Resolver.url(
   new java.net.URL("https://dl.bintray.com/meetup/sbt-plugins/")
 )(Resolver.ivyStylePatterns)
 
-addSbtPlugin("com.meetup" % "sbt-plugins" % "0.3.26")
```
Comment (chenrui333) [inline project/plugins.sbt:6]: relates to https://github.com/meetup/core-best-service.g8/pull/68

File: project/build.properties [hunk 3]
```diff
@@ -1 +1 @@
-sbt.version=0.13.13
```
Comment (chenrui333) [inline project/build.properties:1]: last version before 1.0.0

File: build.sbt [hunk 4]
```diff
@@ -10,7 +10,7 @@ sbtPlugin := true
 
 scalaVersion := appConfiguration.value.provider.scalaProvider.version
 
-libraryDependencies += "io.swagger" % "swagger-codegen" % "2.2.1"
```
Comment (chenrui333) [inline build.sbt:13]: https://github.com/swagger-api/swagger-codegen/releases/tag/v2.3.1

**Review by vadimvlasenko — APPROVED** [review]
lgtm

**Comment by chenrui333** [comment]
Thanks @vadimvlasenko , merging now.

**Pull request merged**